pkg/objstore/objstore_test.go

```diff
@@ -19,12 +19,12 @@ func TestMetricBucket_Close(t *testing.T) {
 	testutil.Equals(t, 7, promtest.CollectAndCount(bkt.opsDuration))
 
 	AcceptanceTest(t, bkt.WithExpectedErrs(bkt.IsObjNotFoundErr))
-	testutil.Equals(t, float64(6), promtest.ToFloat64(bkt.ops.WithLabelValues(OpIter)))
+	testutil.Equals(t, float64(9), promtest.ToFloat64(bkt.ops.WithLabelValues(OpIter)))
```

Comment:
Why this changed? 🤔

Reply:
Because I've changed the `AcceptanceTest` in `pkg/objstore/testing.go` adding more `Upload()` and `Iter()` calls.  This test just calls the `AcceptanceTest()` (see line above).